packages/neuron-wallet/src/services/multisig.ts

```diff
@@ -24,7 +24,7 @@ export default class MultisigService {
         r: multisigConfig.r,
         m: multisigConfig.m,
         n: multisigConfig.n,
-        blake160s: multisigConfig.blake160s,
+        blake160s: multisigConfig.blake160s.toString(),
```

Comment:
After updating to `5.1.6`, the `simple-array` can not compare by `blake160s: multisigConfig.blake160s`?
Do you have any references about this?

Reply:
> After updating to `5.1.6`, the `simple-array` can not compare by `blake160s: multisigConfig.blake160s`? Do you have any references about this?

Actually, I couldn't find an exact same inquiry or documentation in other users' inquiries. After upgrading SQLite's version, this code cannot always retrieve the stored object, and after removing this attribute, other attributes can retrieve the target object. Therefore, after analysis, I think that the query condition may be the cause of the query failure. Because it is a complex object, it may be treated as a new object when passed in for comparison. I may further improve this method by using JSON.stringify to wrap the object.

Reply:
It seems to cause by https://github.com/TryGhost/node-sqlite3/compare/v5.1.4...v5.1.5#diff-dc1d7b24ee601a637b09908305be8f69b96d7a9293643ddc88c435923ff648c8R211.
I have tested if I insert to a simple-array field with `[object Object]`, it can find one item with
```
{
 blake160s: multisigConfig.blake160s,
}
```